stingray/largememory.py

```diff
@@ -378,6 +376,7 @@ def _retrieveDataLC(data_path, chunk_size, offset, raw):
         if offset > times.size:
             raise ValueError((f"Offset cannot be larger than size of array {times.size}"))
 
+        # REVIEW: Is this the right way to go about gtis?
```

Comment:
Took inspiration from `Lightcurve.truncate` is this the correct to go about it?
Please check `gti_new`